spec/requests/api/orchestration_template_spec.rb

```diff
@@ -133,6 +133,14 @@
       run_post(orchestration_templates_url,
                gen_request(:delete, [{'id' => cfn.id}, {'id' => hot.id}]))
 
+      expected = {
+        'results' => [
```

Comment:
You may want to use `a_collection_containing_exactly` here instead of `[]` unless you can guarantee the order